cu-webteam/d8-platform
# Slide refactor

# Description

Brings slide refactor configuration changes in

## Related PRs

N/A

## Todos

- [x ] Tests - tested by creating content and looking at slides and front page

## Deploy Notes

N/A

## Steps to Test or Reproduce

Run configuration import
Sync database from dev to local
Sync files from dev to local
Check front page, which should have a panelized page with a slider
